### PR TITLE
new post docker namespace

### PR DESCRIPTION
**Pull request by davidino**

### CODE REVIEW

**Comment by davidino** [comment]
@razanbilwani can you check it , please ?

File: source/_posts/2016-02-25-docker-security-improvement-username-space.markdown [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+drwxr-xr-x    3 root     root          4096 Nov  5 08:49 installer
+-rw-r-----    1 root     adm           6378 Dec 15 12:18 apport.log.1
+drwxr-x---    2 root     adm           4096 Dec 17 06:04 unattended-upgrades
+-rw-r-----    1 104      adm        1351288 Dec 19 05:56 syslog.2.gz
+-rw-r-----    1 104      adm         402127 Dec 25 06:17 auth.log.1
+-rw-r-----    1 104      adm        8667895 Dec 25 06:17 kern.log.1
+-rw-r-----    1 104      adm         932806 Dec 25 06:21 syslog.1
+```
+
+Notice that the docker deamon is running with no addional options.
```
Comment (odino) [inline source/_posts/2016-02-25-docker-security-improvement-username-space.markdown]: additional

File: source/_posts/2016-02-25-docker-security-improvement-username-space.markdown [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+comments: true
+categories: [ docker ]
+author: David Funaro
+---
+
+{% img center /images/posts/docker-share-volume.png %}
+
+With the 1.10 release, Docker added a huge list of new features. This post will explain about the security update, username spaces.
+
+Prior to version 1.10, running an alpine container mounting an external volume `/var/log/` was done as follows:
```
Comment (razanbilwani) [inline source/_posts/2016-02-25-docker-security-improvement-username-space.markdown]: "mounting an external volume" > "mounted on an external volume"

File: source/_posts/2016-02-25-docker-security-improvement-username-space.markdown [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+```
+
+Notice that the docker deamon is running with no addional options.
+
+```
+➜  ~  ps -ef | grep -i daemon
+root      21636      1  2 15:04 ?        00:00:00 /usr/bin/docker daemon -H fd://
+```
+
+The ownership of the files in the container it is exactly same as the host. The user inside the container as full rights to the files. With this access, the user can even delete the files which will delete the files in the host as well. This is the problem!
```
Comment (razanbilwani) [inline source/_posts/2016-02-25-docker-security-improvement-username-space.markdown]: "it is" > "is"

Comment (razanbilwani) [inline source/_posts/2016-02-25-docker-security-improvement-username-space.markdown]: "as" > "has"

File: source/_posts/2016-02-25-docker-security-improvement-username-space.markdown [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+drwxr-xr-x    3 nobody   nobody        4096 Nov  5 08:49 installer
+-rw-r-----    1 nobody   nobody        6378 Dec 15 12:18 apport.log.1
+drwxr-x---    2 nobody   nobody        4096 Dec 17 06:04 unattended-upgrades
+-rw-r-----    1 nobody   nobody     1351288 Dec 19 05:56 syslog.2.gz
+-rw-r-----    1 nobody   nobody      402127 Dec 25 06:17 auth.log.1
+-rw-r-----    1 nobody   nobody     8667895 Dec 25 06:17 kern.log.1
+-rw-r-----    1 nobody   nobody      932806 Dec 25 06:21 syslog.1
+```
+
+The result is, owner is `nobody` now. Even root user of the container cannot change the files. This is a great security upgrade that everybody was waiting for.
```
Comment (razanbilwani) [inline source/_posts/2016-02-25-docker-security-improvement-username-space.markdown]: "even root" > "even the root"

**Pull request merged**